src/app/components/hris/employees/employee-profile/employee-profile.component.ts

```diff
@@ -145,8 +148,21 @@ export class EmployeeProfileComponent implements OnChanges {
   ngOnDestroy() {
     this.displayEditButtons()
   }
+    
+  // getNewProfile (){
+  //   this.employeeProfile$?.subscribe( profile =>{
+  //    this.BIGemployeeProfile = profile;
+  //     console.log(this.BIGemployeeProfile)
+  //    })
+  //  }
```

Comment:
Should we remove this comment if not going to be used?

Reply:
i can take it out 👍🏻